src/components/label/Label.tsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import type { PropsWithChildren } from 'react';
+
+import { Fieldset, Label as DesignsystemetLabel } from '@digdir/designsystemet-react';
+import { Grid } from '@material-ui/core';
+
+import classes from 'src/components/label/Label.module.css';
+import { LabelContent } from 'src/components/label/LabelContent';
+import { gridBreakpoints } from 'src/utils/formComponentUtils';
+import type { LabelContentProps } from 'src/components/label/LabelContent';
+import type { IGridStyling, ILabelSettings } from 'src/layout/common.generated';
+
+type LabelType = 'legend' | 'span' | 'label';
+
+export type LabelProps = PropsWithChildren<{
+  id: string;
+  renderLabelAs: LabelType;
+  required?: boolean;
+  readOnly?: boolean;
+  labelSettings?: ILabelSettings;
+  grid?: { labelGrid?: IGridStyling };
+  textResourceBindings?: {
+    title?: string;
+    description?: string;
+    help?: string;
+    helpText?: string;
+  };
+}>;
+
+export function Label({
+  id,
+  renderLabelAs,
+  required,
+  readOnly,
+  labelSettings,
+  grid,
+  textResourceBindings,
+  children,
+}: LabelProps) {
+  if (!textResourceBindings?.title) {
+    return <>{children}</>;
+  }
+
+  const labelId = `label-${id}`;
+  const labelContentProps: Omit<LabelContentProps, 'id'> = {
+    label: textResourceBindings.title,
+    description: textResourceBindings.description,
+    helpText: textResourceBindings.help ?? textResourceBindings.helpText,
```

Comment:
Is both `help` and `helpText` needed here? I see that options use `helpText` instead of `help` for some reason but is this component ever used to render options? Options also do not have a `title` but `label` instead, so only including `helpText` here does not really make sense, unless I have missed something of course 😅

Reply:
Also applies to the `LabelProps` type

Reply:
Godt poeng! Hadde bare `helpText` først, men det ble ikke riktig og så fant jeg at det var `help` i stedet fra config i flere filer. Trodde både `help` og `helpText` ble brukt og var tydeligvis for slapp til å faktisk sjekke det 😅